build.gradle

```diff
@@ -7,7 +7,10 @@ allprojects {
     group = 'com.holmsted.gerrit'
 
     dependencies {
-        compile 'com.google.code.findbugs:annotations:3.0.1'
+        implementation 'com.google.code.findbugs:annotations:3.0.1'
+        implementation 'com.google.guava:guava:18.0'
+        implementation 'org.json:json:20140107'
+        implementation 'com.beust:jcommander:1.48'
```

Comment:
Can we keep the added dependencies where they were before? Not all of the projects need to depend on every library.

Separately, if we want to declare the list of dependencies in just one place so we can make sure that each module is using the same version, we could do
```
ext.libs = [
  guava: 'com.google.guava:guava:18.0',
  ...
]
``` 
in this file, and then in other build.gradle files, we could refer to that by saying `implementation libs.guava` or something of that sort.

Reply:
Please make any change you like to this PR, it was just a first draft.

Reply:
@chrisinmtown since you were wondering about the GitHub etiquette elsewhere; the author tends to 'own' their own PRs in most projects I've worked on. I hope this doesn't dampen your enthusiasm, it's great to see these PRs coming from you! 👍

Reply:
Thanks yeah I get it :) but gradle is black magic to me so I'm hoping maybe you'll make the fine adjustments and I'll learn from that.  

Instead of fighting gradle, I'm focusing on extending this tool for HTTPS with as little disruption as possible to existing code.